src/components/query-viewer-pivot-table/controller/query-viewer-pivot-render.tsx

```diff
@@ -285,24 +300,40 @@ export class QueryViewerPivotTableRender {
   }
 
   render() {
-    if (this.serviceResponse == null) {
-      return "";
-    }
-
-    const pivotParameters = this.getPivotTableParms();
+    const pivotParameters = this.getPivotTableParameters();
     const pivotCollection = this.getPivotTableCollection();
 
+    if (this.tableType === QueryViewerOutputType.PivotTable) {
+      return (
+        <Host>
+          {this.serviceResponse != null && (
+            <gx-query-viewer-pivot
+              pivotCollection={pivotCollection}
+              pivotParameters={pivotParameters}
+              pageDataForPivotTable={this.pageDataForPivotTable}
+              attributeValuesForPivotTableXml={
+                this.attributeValuesForPivotTableXml
+              }
+              calculatePivottableDataXml={this.calculatePivottableDataXml}
+              getPivottableDataSyncXml={this.getPivottableDataSyncXml}
+              ref={el => (this.pivotRef = el)}
+            ></gx-query-viewer-pivot>
+          )}
```

Comment:
Complements the previous comment. It is no longer needed to check the `serviceResponse`.

```suggestion
          <gx-query-viewer-pivot
            pivotCollection={pivotCollection}
            pivotParameters={pivotParameters}
            pageDataForPivotTable={this.pageDataForPivotTable}
            attributeValuesForPivotTableXml={
              this.attributeValuesForPivotTableXml
            }
            calculatePivottableDataXml={this.calculatePivottableDataXml}
            getPivottableDataSyncXml={this.getPivottableDataSyncXml}
            ref={el => (this.pivotRef = el)}
          ></gx-query-viewer-pivot>
```